release/README.md

```diff
@@ -61,13 +74,13 @@ $ githubctl --token_file=<github token file> \
 # The previous command triggers rebuild and retagging on pilot, mixer and auth.
 # Wait for them to finish, then create an update PR in istio/istio
 $ githubctl --token_file=<github token file> \
```

Comment:
can you specify which directory githubctl must be invoked if there is a restriction?

Reply:
There is no restriction I added an alias to fix the documentation.